run_scripts/test.py

```diff
@@ -83,14 +83,14 @@
     full_alchemical_system = factory.createPerturbedSystem(AlchemicalState(lambda_electrostatics=1, lambda_sterics=1))
     alchemical_system = factory.createPerturbedSystem()
 
- 
+
 if 1: #if cluster test system
```

Comment:
`if 0` and `if 1` are sort of horrible hacks. Can't we at least use a variable name like `cluster_test_system = True` and `if cluster_test_system` or some such?